google-cloud-spanner/src/test/java/com/google/cloud/spanner/SpanTest.java

```diff
@@ -201,7 +201,7 @@ public void setUp() throws Exception {
             .setSessionPoolOption(
                 SessionPoolOptions.newBuilder()
                     .setMinSessions(2)
-                    .setWaitForMinSessions(Duration.ofSeconds(5))
+                    .setWaitForMinSessions(Duration.ofSeconds(10))
```

Comment:
Is this needed in order to prevent it from being flaky? (It probably doesn't hurt, but it seems like a surprising change)

Reply:
I think both `SpanTest` and `OpenTelemetrySpanTest` exude a flaky behavior today irrespective of multiplexed sessions. What I saw is increasing this time reduces the flakiness.